src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewDeveloperSteps.java

```diff
@@ -0,0 +1,80 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import gov.healthit.chpl.aqa.pageObjects.ViewDeveloperPage;
+
+/**
+ * Class ViewDeveloperSteps definition.
+ */
+public class ViewDeveloperSteps extends Base {
+
+    /**
+     * Get user to the View Developer Page.
+     * @param developerId is Developer ID
+     **/
+    @Given("^I navigate to the developer edit page for developer \"([^\"]*)\"$")
+    public void editDeveloperPage(final String developerId) {
+        getDriver().get(getUrl() + "#/organizations/developers/" + developerId);
+        WebDriverWait wait = new WebDriverWait(getDriver(), TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(ViewDeveloperPage.viewDeveloperPageTitle(getDriver())));
+    }
+
+    /**
+     * Edit a developer.
+     * @param developerId is the developer id passed in page objects
+     **/
+    @And("^I edit for developer \"(.*)\"$")
+    public void editDeveloper(final String developerId) {
+        ViewDeveloperPage.editDeveloperLink(getDriver(), developerId).click();
```

Comment:
Could you call the below method clickEditLinkOfDeveloper(developerId) here instead of writing this line again?